common.py

```diff
@@ -135,21 +135,47 @@ class IPRouteAddressEntry:
     addr_info: list[IPRouteAddressInfoEntry]
 
 
+def _json_load(jstr: str, *, strict_parsing: bool = False) -> list[typing.Any]:
```

Comment:
Let's rename this function to what it does: `load_json_list`. Also, instead of having a boolean flag to toggle part of the function, let's make it return an optional list, and have the callee handle the case when it returns None.

Reply:
this function is only an implementation detail of the 3 callers that it has. The point is to move common code out, to avoid re-implementing it.

Returning and `Optional[list]` and let each caller handle that again, means that `_json_load()` does less, and callers will reimplement such error handling. I'd rather leave this.

I will rename.